.github/workflows/e2e.yml

```diff
@@ -152,10 +152,50 @@ jobs:
       - name: Run e2e test
         run: timeout 2400 bats "e2e/$E2E_TEST"
 
+  ui_test:
+    runs-on: ${{ matrix.os }}
+    needs: [build_dfx]
+    strategy:
+      matrix:
+        os: [macos-12, ubuntu-20.04, ubuntu-22.04]
+    steps:
+      - name: Checking out repo
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
Thanks!

I usually use it to mark objects as private. Probably, in this script they are a bit exaggerated, and we can also consider to remove them.